packages/cli-e2e/__tests__/atomic.specs.os.ts

```diff
@@ -92,7 +94,7 @@ describe('ui:create:atomic', () => {
       .do(answerPrompt(`y${EOL}`))
       .until(buildTerminalExitPromise);
 
-    return {stderr};
+    return {stderr: output};
```

Comment:
stdout in stderr?

Reply:
derp, i'll rename the var output.
Long story short, when using the published version of the CLI, everything ends up in stdout